app/models/week.rb

```diff
@@ -34,6 +49,51 @@ def actual?
     (Date.today-self.start_at).to_i.between?(0,6)
   end
 
+  def free_los user_id
+    sql = '(SELECT DISTINCT learning_objects.* FROM learning_objects
```

Comment:
Tieto SQLka su peklo. Beriem, ze nejake nam usli uz do sucasneho codebase, ked sme narychlo robili MVP (minimum viable product), ale uz to tu nerobme. Dalsie zlo je, ze sa tu opakuju v miernych obmenach pod sebou, takze to porusuje DRY.

Skus to prepisat do ARELu. Ako na to pozeram, nemal by to byt problem. Niektore tie veci maju aj definicie typu `has_many` a `belongs_to` v modeloch (aj v tomto tu hore), ak nieco chyba, prosim, dopln.